components/landing/hero/index.ts

```diff
@@ -1,3 +1,3 @@
-import Hero from './Hero';
+import Hero from './hero';
```

Comment:
This should be ./Hero, the files starts with a capital H

Reply:
✔️